auth.go

```diff
@@ -444,6 +452,39 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 	return sessionkey, token, nil
 }
 
+func auditUserAction(identity, password, actionDescription string) error {
```

Comment:
I don't think this function should return an error - primarily because the caller will never DO anything with that error. So rather have a wrapper function, which calls the Auditor's `AuditUserAction` function, and then logs the failure there.